src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1453,7 +1454,7 @@ private class Parameters
             public Action<IOperationProgress> OnProgress;
         }
 
-        public async Task<IOperationResult> RevertRevisions(DateTime before, TimeSpan window, Action<IOperationProgress> onProgress, OperationCancelToken token)
+        public async Task<IOperationResult> RevertRevisions(DateTime before, TimeSpan window, Action<IOperationProgress> onProgress, OperationCancelToken token, string collection = null)
```

Comment:
we want to support multiple collections, pass HashSet<string> here, remember that we are not case sensitive here

Reply:
Do we need to distinguish between: empty set of collection & all collections?

Reply:
If ```collections``` value is null, it will be considered as 'all collections'